src/Adapter/Product/Combination/QueryHandler/GetCombinationForEditingHandler.php

```diff
@@ -77,12 +86,35 @@ public function handle(GetCombinationForEditing $query): CombinationForEditing
         $combination = $this->combinationRepository->get($query->getCombinationId());
 
         return new CombinationForEditing(
+            $this->getCombinationName($query->getCombinationId()),
             $this->getDetails($combination),
             $this->getPrices($combination),
             $this->getStock($combination)
         );
     }
 
+    /**
+     * @param CombinationId $combinationId
+     *
+     * @return string
+     */
+    private function getCombinationName(CombinationId $combinationId): string
+    {
+        $attributesInformation = $this->combinationRepository->getAttributesInfoByCombinationIds(
+            [$combinationId->getValue()],
+            new LanguageId($this->contextLanguageId)
+        );
+        $attributes = $attributesInformation[$combinationId->getValue()];
+
+        return implode(', ', array_map(function ($attribute) {
+            return sprintf(
+                '%s - %s',
+                $attribute['attribute_group_name'],
+                $attribute['attribute_name']
+            );
+        }, $attributes));
```

Comment:
@MatShir 
it's not related to this PR, but i was wondering if we would give people the option to choose the separator for the declension names ? `-` vs `:`
Ex.: 
`Coussin Coloris - beige, Dimension - 30x50 cm` **vs** `Coussin Coloris : beige, Dimension : 30x50 cm`

Reply:
Approved !!!!

Reply:
I have made multiple times this feedback ;)

Reply:
I believe we discussed this with @matks some time ago This is not configurable for now but I guess it could (should?) be Although I don't think it's in the scope of this PR

Reply:
It needs to be handled in a dedicated PR because if I remember correctly we have some js code that "parses" the combination name sometimes so if the separator suddenly becomes different it won't work any more So we need to this the right way and modify all the impacted code with the new configurable separator

I don't remember where exactly it was done though, maybe it's in the legacy product page, maybe in order page 🤷‍♂️  If it's only in legacy product page since it will be removed in 1.7.9 maybe we can simply ignore it.

Reply:
Yes, I have fixed that in 177 for combinations product.

Reply:
You mean it's now configurable? Or you fixed it with hard values?

Reply:
Configurable